tests/MongoDB.Driver.Core.Tests/Specifications/initial-dns-seedlist-discovery/tests/one-txt-record-multiple-strings.json

```diff
@@ -10,6 +10,6 @@
   ],
   "options": {
     "replicaSet": "repl0",
-    "ssl": true
+    "tls": true
```

Comment:
@rstam, should we create a spec ticket for these changes? (Replacing `"ssl":true` on `"tls" : true` in `initial-dns-seedlist-discovery` folder)?

Reply:
Probably?

Or a test runner could treat `ssl` and `tls` as equivalent.

Reply:
Either way we want to be in sync with the spec repo.